pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "bradbot"
+version = "0.1.0"
+description = ""
+authors = ["brad90four <42116429+brad90four@users.noreply.github.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+nextcord = "^2.0.0-alpha.6"
+loguru = "^0.5.3"
+python-dotenv = "^0.19.2"
+aiohttp = "^3.8.1"
+aiodns = "^3.0.0"
+pre-commit = "^2.16.0"
+flake8 = "^4.0.1"
+black = "^21.12b0"
+isort = "^5.10.1"
```

Comment:
What's the reason for including the linting deps when linting is handled in pre-commit?